### PR TITLE
fix(PP-804): do not show an error when the address file is missing

### PR DESCRIPTION
**Pull request by antomor**
## What

- do not raise an error when the address file is missing during the contracts deployment

## Why

- the error can be misleading and the developer could think that the deployment wasn't successful

### CODE REVIEW

File: tasks/deploy.ts [hunk 1]
```diff
@@ -32,8 +33,15 @@ export const updateConfig = (
     throw new Error('Unknown Chain Id');
   }
 
+  let existingConfig = {};
+  try {
+    existingConfig = parseJsonFile<AddressesConfig>(ADDRESS_FILE);
+  } catch (error) {
+    console.log(`Previous configuration not found at: "${ADDRESS_FILE}"`);
+  }
```
Comment (jurajpiar) [inline tasks/deploy.ts:41]:
```suggestion
  const existingConfig = await Promise.resolve(parseJsonFile<AddressesConfig>(ADDRESS_FILE)).catch(e => {
  console.log(`Previous configuration not found at: "${ADDRESS_FILE}"`);
  })
```
and changing `updateConfig` signature to `async` and calling it in `‎test/tasks/deploy.test.ts‎` (the only found usage) with `await`.

**Review by jurajpiar — APPROVED** [review]
LG, just would promisify the try/catch

File: tasks/deploy.ts (hunk 1 above)
Comment (antomor) [reply tasks/deploy.ts]: Updated to replace the try/catch with a promise

**Comment by franciscotobar** [comment]
I tried it and it works properly :)

**Pull request merged**